jest/jest.d.ts

```diff
@@ -1,126 +1,88 @@
+declare function xit(name: string, fn: () => any): void;
 
 declare namespace jest {
-    function addMatchers(matchers: CustomMatcherFactories): void;
-    function autoMockOff(): void;
-    function autoMockOn(): void;
-    function clearAllTimers(): void;
-    function currentTestPath(): string;
-    function disableAutomock(): void;
-    function fn<T>(implementation?: Function): Mock<T>;
-    function dontMock(moduleName: string): void;
-    function genMockFromModule<T>(moduleName: string): Mock<T>;
-    function mock(moduleName: string, factory?: Function): void;
-    function runAllTicks(): void;
-    function runAllTimers(): void;
-    function runOnlyPendingTimers(): void;
-    function setMock<T>(moduleName: string, moduleExports: T): void;
-    function unmock(moduleName: string): void;
-
-    interface EmptyFunction {
-        (): void;
-    }
-
     interface Matchers {
+        lastCalledWith(...args: any[]): boolean;
         not: Matchers;
-        toThrow(expected?: any): boolean;
-        toThrowError(expected?: any): boolean;
         toBe(expected: any): boolean;
-        toEqual(expected: any): boolean;
+        toBeCalled(): boolean;
+        toBeCalledWith(...args: any[]): boolean;
+        toBeCloseTo(expected: number, delta: number): boolean;
+        toBeDefined(): boolean;
         toBeFalsy(): boolean;
-        toBeTruthy(): boolean;
+        toBeGreaterThan(expected: number): boolean;
+        toBeGreaterThanOrEqual(expected: number): boolean;
+        toBeLessThan(expected: number): boolean;
+        toBeLessThanOrEqual(expected: number): boolean;
         toBeNull(): boolean;
-        toBeDefined(): boolean;
+        toBeTruthy(): boolean;
         toBeUndefined(): boolean;
-        toMatch(expected: RegExp): boolean;
         toContain(expected: string): boolean;
-        toBeCloseTo(expected: number, delta: number): boolean;
-        toBeGreaterThan(expected: number): boolean;
-        toBeLessThan(expected: number): boolean;
-        toBeCalled(): boolean;
-        toBeCalledWith(...args: any[]): boolean;
-        lastCalledWith(...args: any[]): boolean;
+        toEqual(expected: any): boolean;
+        toMatch(expected: RegExp): boolean;
+        toMatchSnapshot(): boolean;
+        toThrow(): boolean;
+        toThrowError(expected: string | RegExp): boolean;
+        toThrowError<TFunction extends Function>(expected: TFunction): boolean;
     }
-
-    interface It {
-        (name: string, fn: EmptyFunction): void;
-        only(name: string, fn: EmptyFunction): void;
+    
+    interface MockContext<T> {
+        calls: any[][];
+        instances: T[];
     }
 
     interface Mock<T> {
         new (): T;
-        (...args: any[]): any; // TODO please fix this line! added for TypeScript 1.1.0-1 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/2932
+        (...args: any[]): any; // Making Mock Callable and fixing: Value of type 'Mock<T>' is not callable.
```

Comment:
If we have a test case for this I don't think the comment is necessary. It makes sense for a type without a call signature to error if you try to call it 😁